src/components/screens/Cadastros/EditaisContratos/ContratosRelacionados.jsx

```diff
@@ -41,7 +42,9 @@ class ContratosRelacionados extends Component {
   obtemDataInicial(keyVigencia, indiceForm) {
     if (keyVigencia === 0) {
       return moment(
-        this.props.contratos_relacionados[indiceForm]["data_proposta"],
+        this.props.atualizacao
+          ? this.props.contratos_relacionados[indiceForm]["data_proposta"]
```

Comment:
const {contratos} = this.props.contratos_relacionados

contratos[indiceForm]["data_proposta"]

Reply:
ok